.github/workflows/verify.yaml

```diff
@@ -127,3 +127,22 @@ jobs:
 
       - name: Run E2E tests
         run: ./gradlew :edc-tests:runtime:build test -DincludeTags="EndToEndTest"
+
+  postgres-end-to-end-tests:
```

Comment:
nit: rename to `postgres-tests`. How would you feel about running this in the `integration-test` job instead as a separate job?